packages/seacas/libraries/ioss/src/main/io_shell.C

```diff
@@ -87,8 +90,13 @@ int main(int argc, char *argv[])
 #endif
 
   double begin = Ioss::Utils::timer();
+
   try {
-    file_copy(interFace, rank);
+    if( interFace.compare == false ) {
```

Comment:
I would reverse:
```
if (interFace.compare) {
   file_compare(interFace, rank);
}
else {
   file_copy(interFace, rank);
}
```
Or maybe `if (!interFace.compare)` and leave current ordering.